src/services/cherryPickService.ts

```diff
@@ -146,18 +147,21 @@ export async function CreatePullRequestAsync(
       prSearchCriteria
     );
 
+    const pullRquestURL = `${
+      pullRequestContext.repository.webUrl
+    }/pullrequest/${pullRequestContext.pullRequestId}_a=overview`;
```

Comment:
Do we need to append the `_a=overview` or would the link work without it?

Reply:
Works without it! Will update